tests/toolkit/res_buf_tests.h

```diff
@@ -63,6 +65,56 @@ class ResBufTest : public ::testing::Test {
   }
 };
 
+class MaterialBufTest : public ::testing::Test {
+ protected:
+
+  ResBuf<Material> mat_store_;  // default constructed mat store
+  ResBuf<Material> bulk_store_ = ResBuf<Material>(true);
+
+  Nuc sr89_, fe59_;
+  Material::Ptr mat1_, mat2_, mat3_;
+  Composition::Ptr test_comp1_, test_comp2_, test_comp3_;
+
+  double cap_;
+
+  virtual void SetUp() {
+    try {
+
+      sr89_ = 380890000;
+      fe59_ = 260590000;
+
+      CompMap v, w;
+      v[sr89_] = 1;
+      test_comp1_ = Composition::CreateFromMass(v);
+
+      w[fe59_] = 2;
+      test_comp2_ = Composition::CreateFromMass(w);
+
+      w[sr89_] = 1;
+      test_comp3_ = Composition::CreateFromMass(w);
+
+      mat1_ = Material::CreateUntracked(5 * units::g, test_comp1_);
+      mat2_ = Material::CreateUntracked(5 * units::g, test_comp2_);
+      mat3_ = Material::CreateUntracked(5 * units::g, test_comp1_);
```

Comment:
Is this supposed to be 
```suggestion
      mat3_ = Material::CreateUntracked(5 * units::g, test_comp3_);
```
? If not, why have a `test_comp3_`